pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/DistinctCountThetaSketchAggregationFunction.java

```diff
@@ -387,12 +386,17 @@ public void aggregateGroupByMV(int length, int[][] groupKeysArray, GroupByResult
   public Map<String, Sketch> extractAggregationResult(AggregationResultHolder aggregationResultHolder) {
     Map<Predicate, Union> unionMap = aggregationResultHolder.getResult();
     if (unionMap == null || unionMap.isEmpty()) {
-      return Collections.emptyMap();
+      return null;
```

Comment:
Is the caller already good with null? I mean it doesn't have to be changed to handle null return from this function?

Reply:
Yes, see merge(), there's a null check there. I had added `Collections.emptyMap()` in the previous PR, only to realize that it could be null.

Reply:
On second thought, perhaps emptyMap() is better to avoid null checks in callers. Updated.